src/benchmarks/pmem_flush.c

```diff
@@ -62,7 +62,7 @@ align_addr(char *addr, uintptr_t align)
  * align_len -- increase len by the amount we gain when we round addr down
  */
 static size_t
-align_len(size_t len, char *addr, uintptr_t align)
+align_len(size_t len, void *addr, uintptr_t align)
```

Comment:
The reason that this is char * and not void * is that void pointer arithmetic is a GNU extension and thus non-standard. Leave it as char * please. There are compiler warnings that will complain about this.